test-harness/scenarios/parameter-description-parameters.oas2.scenario

```diff
@@ -0,0 +1,34 @@
+====test====
```

Comment:
Consider adding a positive test checking whether we don't return warnings if descriptions are provided.